build.gradle

```diff
@@ -2,7 +2,8 @@ apply plugin: 'java'
 apply plugin: 'maven'
 
 group = 'io.curity.identityserver.plugin'
-version = '2.2.2'
+version = '1.0.0'
```

Comment:
could cause confusion to go down in version numbers, so I'd suggest to not.

Reply:
We haven't published a release on this plugin though

Reply:
What version should i put ?

Reply:
ah, OK. Then, 1.0.0 is OK I suppose. Otherwise, I was thinking 2.3.0